pkg/virt-handler/vm.go

```diff
@@ -2560,6 +2575,18 @@ func (d *VirtualMachineController) checkVolumesForMigration(vmi *v1.VirtualMachi
 	return
 }
 
+func (d *VirtualMachineController) isVMIPausedDuringMigration(vmi *v1.VirtualMachineInstance) bool {
+
+	if vmi.Status.MigrationState != nil &&
```

Comment:
you can simplify this and return the condition itself
```golang
return  vmi.Status.MigrationState != nil && 		
             vmi.Status.MigrationState.Mode == v1.MigrationPaused &&
             !vmi.Status.MigrationState.
```

Reply:
Thanks!